words/compile.asm

```diff
@@ -268,6 +268,10 @@ _not_uf:        clc                     ; C=0 means it isn't a UF check
 ; We have have various utility routines here for compiling a word in Y/A
 ; and a single byte in A.
 
+; TODO for all of these we could potentially avoid jmp (and NN) and
```

Comment:
i'll work on this in a different PR, but probably could get rid of most `JMP`s to keep NN for more user words